### PR TITLE
Make allowance component style 

### PR DESCRIPTION
**Pull request by patjanawat**
Make the component style as the same distribution configs (**Closes**).

Related to: https://github.com/mesg-foundation/atd/issues/49

### CODE REVIEW

File: frontend/components/admin/allowance.vue [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
-        <v-btn color="primary" dark class="mb-2" @click="newItem()">New Item</v-btn>
-      </template> -->
+    <v-dialog v-model="dialog" persistent max-width="500px">
       <v-card>
-        <v-card-title>
-          <span class="headline">{{ formTitle }}</span>
-        </v-card-title>
-
+        <v-toolbar flat color="white">
+          <v-toolbar-title>Allowance Configs</v-toolbar-title>
```
Comment (NicolasMahe) [inline frontend/components/admin/allowance.vue]:
```suggestion
          <v-toolbar-title>Edit Allowance</v-toolbar-title>
```

File: frontend/components/admin/allowance.vue [hunk 2]
```diff
@@ -42,88 +43,70 @@
...
+      ]
     };
   },
   computed: {
-    formTitle() {
-      return this.editedIndex === -1 ? "New Item" : "Edit Item";
-    },
-    locked(){
-      return this.editedIndex === -1? false: true
+    locked() {
```
Comment (NicolasMahe) [inline frontend/components/admin/allowance.vue]: I think you can remove this property "locked".
The allowances can only be edited, not added. So you can simplify even more the logic.

File: frontend/components/admin/allowance.vue [hunk 3]
```diff
@@ -1,35 +1,36 @@
...
-              <v-flex xs12>
+              <v-flex xs12 md12>
                 <v-text-field :disabled="locked" v-model="editedItem.address" label="Address"></v-text-field>
               </v-flex>
-              <v-flex xs12>
-                <v-text-field v-model="editedItem.allowance" label="Value"></v-text-field>
+              <v-flex xs12 md12>
+                <v-text-field
+                  v-model="editedItem.allowance"
+                  type="text"
```
Comment (NicolasMahe) [inline frontend/components/admin/allowance.vue]: Maybe you can use type number:
```suggestion
                  type="number"
```

File: frontend/components/admin/allowance.vue (hunk 1 above)
Comment (patjanawat) [reply frontend/components/admin/allowance.vue]: Done.

File: frontend/components/admin/allowance.vue (hunk 2 above)
Comment (patjanawat) [reply frontend/components/admin/allowance.vue]: Done.

File: frontend/components/admin/allowance.vue (hunk 3 above)
Comment (patjanawat) [reply frontend/components/admin/allowance.vue]: Done.

**Pull request merged**